src/_posts/2021-02-11-cass-changelog_4.markdown

```diff
@@ -0,0 +1,101 @@
+
+We’re curating a list of JIRA tickets this month, which will be labeled as `gsoc2021`. This will make them visible in the [Jira issue tracker](https://issues.apache.org/jira/browse/SYNAPSE-1125?jql=labels%20%3D%20gsoc2021) for participants to see and connect with mentors. 
+
+If you would like to volunteer to be a mentor for a GSoC project, please tag the respective JIRA ticket with the `mentor` label. Non-committers can volunteer to be a mentor as long as there is a committer as co-mentor. Projects can be mentored by one or more co-mentors. 
+
+Thanks to Paulo Motta for proposing the idea and getting the ticket list going.
+
+#### **Added**
+
+Apache Zeppelin [0.9.0](http://zeppelin.apache.org/download.html) was released on January 15. Zeppelin is a collaborative data analytics and visualization tool for distributed, general-purpose data processing system, which supports Apache Cassandra and others. The release notes for the Cassandra CQL Interpreter are available [here](http://zeppelin.apache.org/docs/0.9.0/interpreter/cassandra.html).
+
+#### **Changed**
+
+For the GA of Apache Cassandra 4.0, any claim of support for Python 2 will be dropped from update documentation. We will also introduce a warning when running in Python 2.7. Support for Python 3 will be backported to at least 3.11, due to existing tickets, but we will undertake the work needed to make packaging and internal tooling support Python 3.
+
+#### **Changed**
+
+The Kubernetes SIG is discussing how to encourage more participation and to structure SIG meetings around updates on Kubernetes and Cassandra. We also intend to invite other projects (like OpenEDS, Prometheus, and others) to discuss how we can make Cassandra and Kubernetes better. As well as updates, the group discussed handling large-scale backups inside Kubernetes and using S3 APIs to store images. [Watch here](https://www.youtube.com/watch?v=X5mEgFquIoo).
+
+[![kubernetes-sig-meeting-2021-01-14](http://img.youtube.com/vi/X5mEgFquIoo/0.jpg)](http://www.youtube.com/watch?v=X5mEgFquIoo "")
+
+
+## **User Space**
+
+#### **Backblaze**
+
+“Backblaze uses Apache Cassandra, a high-performance, scalable distributed database to help manage hundreds of petabytes of data.” - [Andy Klein](https://www.backblaze.com/blog/wide-partitions-in-apache-cassandra-3-11/)
+
+#### **Witfoo**
+
+Witfoo uses Cassandra for big data needs in cybersecurity operations. In response to the recent licensing changes at Elastic, Witfoo decided to blog about its journey away from Elastic to Apache Cassandra in 2019. - [Witfoo.com](https://www.witfoo.com/blog/our-move-from-elastic-to-cassandra/)
+
+_Do you have a Cassandra case study to share? Email [cassandra@constantia.io](mailto:cassandra@constantia.io)._
+
+## **In the News**
+
+The New Stack: [What Is Data Management in the Kubernetes Age?
+](https://thenewstack.io/what-is-data-management-in-the-kubernetes-age/)
```

Comment:
A carriage has been added here which breaks the link. This line should be concatenated with the previous line so that is becomes.

```
The New Stack: [What Is Data Management in the Kubernetes Age?](https://thenewstack.io/what-is-data-management-in-the-kubernetes-age/)
```

Reply:
Corrected 👍